mlir/include/mlir/Dialect/Linalg/Transforms/Transforms.h

```diff
@@ -1616,6 +1616,12 @@ void populateSplitReductionPattern(
     const ControlSplitReductionFn &controlSplitReductionFn,
     bool useAlloc = false);
 
+/// Create a TransferReadOp from `source` with static shape `readShape`. If the
```

Comment:
Can you update the comment and describe `doMasking`. (Maybe I'd also rename it to just `masking`.)

Reply:
Could you also include a comment on indices? These are hard-coded to be 0 - that's worth documenting.

> (Maybe I'd also rename it to just masking.)

This is a bit of bikesheding and I don't mind that much ... But I do like when a `bool` variable includes a verb so that it's effectively a question with a binary answer (i.e. "yes"/"no") :) Personally I'd use `enableMasking` instead. Again, don't really mind 😅